### PR TITLE
Minor fixes to docs

### PR DESCRIPTION
**Pull request by natorojr**

### CODE REVIEW

File: UPGRADE_GUIDE.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ active when the index route is active.
 </Route>
 
 // will be active only when home is active, not when about is active
-<IndexLink to="/">Home</Link>
+<Link to="/">Home</Link>
```
Comment (mjackson) [inline UPGRADE_GUIDE.md]: This change is not accurate. You need to use an `IndexLink` to link directly to an `IndexRoute`.

Comment (natorojr) [inline UPGRADE_GUIDE.md]: Fixed.

Justed noticed docs/basics/IndexRoutes.md (which explains IndexLink) is not in the TOC.

Should I add it? Or is this page still a WIP?

**Comment by mjackson** [comment]
Thanks, @natorojr! Can you please rebase on latest master?

**Comment by ryanflorence** [comment]
Thanks!

though, those semi-colons will probably be going away sometime soon :P

**Pull request merged**